style.css

```diff
@@ -78,8 +83,187 @@ body {
   color: #505f79;
 }
 
+li {
+  list-style-type: none;
+}
+
 .social-icons li {
   list-style-type: none;
   padding: 0;
   margin: 0;
 }
+
+/* works section */
+
+.works {
+  display: grid;
+  gap: 25vw;
+  padding: 0 1vw;
+  margin-top: 25vw;
+  margin-bottom: 114px;
+}
+
+.card {
+  width: 90vw;
+  margin: auto;
+  background-color: white;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 1rem;
+  -webkit-border-radius: 1rem;
+  -moz-border-radius: 1rem;
+  -ms-border-radius: 1rem;
+  -o-border-radius: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 4vw;
+  padding-bottom: 5vw;
+}
+
```

Comment:
## Styling ✨✨
- [ ] **Kudos on making your web page responsive 👏 👏** However, as suggested by the previous reviewer, it would be better to replicate the [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=wMKvp4oyMqZ6YudA-0) as much as possible by making some adjustments to the typography ```(font-size, font-weight, spacing)``` to replicate the chosen Figma design 😎😎 
   
  **_You may kindly try to match the following 👇_**
    
   ![image](https://user-images.githubusercontent.com/102757126/235139594-a39d0ffc-38de-4629-8fcf-94fbbf4d603e.png)

- [ ] It would be better also to make a small adjustment to the `margin` to replicate the Figma design as currently the `margin-top` seems to be high compared to that of the Figma design 😊

![image](https://user-images.githubusercontent.com/102757126/235141018-721ac380-b6b4-4c01-a93a-57b5b2bc38ac.png)